index.html

```diff
@@ -48,5 +51,187 @@ <h2 class="prof">I’m a Software Developer</h2>
         </div>
     </div>
 </section>
+
+
+<!-- Recent Works -->
+
+<section id="works-section">
+
+
+  
+  <h3 class="hr">My Recent Work <hr> </h3>
+  
+
+
+<div class="box-container">
+  <div class="left-section">
+    <img src="./img/main (1).png" alt="Recent Project">
+    <div  class="first-desc">
+      <h3>Multi-Post Stories</h3>
+      <P>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</P>
```

Comment:
- [ ] You have a horizontal scrollbar on your screen. Kindly fix the issue. :+1: 


https://github.com/Xghayor/Portfolio/assets/70483835/0f1e07a6-745c-4382-83c0-7ddee6cbae78